app/services/fees/create_pay_in_advance_service.rb

```diff
@@ -154,20 +154,22 @@ def deliver_webhooks
       result.fees.each { |f| SendWebhookJob.perform_later('fee.created', f) }
     end
 
-    def update_event_metadata(aggregation_result:)
-      unless aggregation_result.current_aggregation.nil?
-        event.metadata['current_aggregation'] = aggregation_result.current_aggregation
-      end
-
-      unless aggregation_result.max_aggregation.nil?
-        event.metadata['max_aggregation'] = aggregation_result.max_aggregation
-      end
-
-      unless aggregation_result.max_aggregation_with_proration.nil?
-        event.metadata['max_aggregation_with_proration'] = aggregation_result.max_aggregation_with_proration
-      end
-
-      event.save!
+    def cache_aggregation_result(aggregation_result:, group:)
+      return unless aggregation_result.current_aggregation.present? ||
+        aggregation_result.max_aggregation.present? ||
```

Comment:
Layout/MultilineOperationIndentation: Align the operands of a condition in an `unless` statement spanning multiple lines.